scripts/msct_register.py

```diff
@@ -58,13 +58,13 @@ def register_slicewise(fname_src,
     # Calculate displacement
     if paramreg.algo == 'centermass':
         # translation of center of mass between source and destination in voxel space
-        register2d_centermassrot('src.nii', 'dest.nii', fname_warp=warp_forward_out, fname_warp_inv=warp_inverse_out, rot=0, poly=int(paramreg.poly), path_qc=path_qc, verbose=verbose)
+        register2d_centermassrot('src.nii', 'dest.nii', fname_warp=warp_forward_out, fname_warp_inv=warp_inverse_out, rot=0, poly=int(paramreg.poly), qc_path=qc_path, verbose=verbose)
```

Comment:
why the decision to interchange this variable (same throughout this PR)? it was in line with [our standards](https://github.com/neuropoly/spinalcordtoolbox/wiki/variables#variables) (which could be revisited if needed):

Reply:
I flipped a coin... I'll change everything to `path_qc`, no problem.

Reply:
(done)